README.md

```diff
@@ -661,6 +661,7 @@ Get a list of transactions — ``remme transaction get-list``:
 | limit       | Integer|  No       | Maximum amount of transactions to return.              |
 | head        | String |  No       | Block identifier to get a list of transactions from.   |
 | reverse     | Bool   |  No       | Parameter to reverse result.                           |
+| ids-only    | Bool   |  No       | The flag to get a list of transaction's identifiers only. |
```

Comment:
`transaction's identifiers` > `transactions' identifiers`. It is a list of identifiers of the list of transactions.

Reply:
@dmytrostriletskyi, done.